proca/lib/proca_web/schema/target_types.ex

```diff
@@ -0,0 +1,45 @@
+defmodule ProcaWeb.Schema.TargetTypes do
+  @moduledoc """
+  Schema decleration for Target and TargetEmail types
+  """
+
+  use Absinthe.Schema.Notation
+  alias ProcaWeb.Resolvers
+  alias ProcaWeb.Resolvers.Authorized
+
+  input_object :target_email do
+    field :email, non_null(:string)
+  end
+
+  input_object :target_input do
+    field :name, non_null(:string)
+    field :area, :string
+    field :campaign_id, non_null(:integer)
+    field :external_id, non_null(:string)
+    field :fields, :json
+    field :emails, list_of(:target_email)
+  end
+
+  object :target do
+    field :id, non_null(:string)
+    field :name, non_null(:string)
+    field :area, :string
+    field :fields, :json
+  end
+
+  object :target_mutations do
+    field :upsert_targets, type: non_null(list_of(:target)) do
+      arg :targets, non_null(list_of(:target_input))
+
+      resolve(&Resolvers.Target.upsert_targets/3)
+    end
+  end
+
+  object :target_queries do
+    field :targets, non_null(list_of(:target)) do
```

Comment:
can the targets query be a field of private_campaign and public_campaign ( but the targets under private_campaign should show all fields of target, and targets under private_campaign just a public subset).

Reply:
So the private campaign allows showing emails as well?

Reply:
Yeah private is for dashboard for example, if we would like to list the emails somewhere